lib/Charts/ChartData.js

```diff
@@ -72,6 +73,17 @@ var ChartData = function(points, parameters) {
      * @type {String}
      */
     this.yAxisMax = parameters.yAxisMax;
+    
+    /**
+     * Chart type. If you want these points to be rendered with a certain way. Leave empty for auto detection.
+     * @type {String}
+     */
+    this.type = parameters.type;
+    
+    /**
+     * Chart Renderer Object corresponding to type. Used internally for rendering the chart.
+     */
+    this.renderer = undefined;
```

Comment:
Putting the renderer into a class called ChartData feels iffy - is there another way to do it?